emoncms/emoncms
# added requesting ip address to cookie not present log

adds requesting ip address to error log when cookie was not sent with incoming request.

writes to the log file in this format:
```
2020-01-20 10:26:35.178|INFO|rememberme_model.php|getCookieValues: not present for: 127.0.0.1
```
related to #1499 

more testing required on different hardware and OSes. Please comment here if this works or fails on your particular setup.

### TESTING:
the IP address will be written to the log file (`emoncms.log`) as an `INFO` message (log `level=1`)  if an attempt to login is made by a browser without a cookie set for your install of EmonCMS:-

1. set the log level to `1`(info)  in `settings.ini` to write the IP address to the logfile
2. clear cookies for the domain. [screenshot below](#clearing-cookies)
3. keep logfile changes open in a terminal window
```
$ tail -f /emoncms/emoncms.log
```
4. go to the login page the login page `http://localhost/emoncms/user/login` and watch for the changes in `emoncms.log`. The IP address should be shown as an `INFO` message.

### Clearing Cookies
[screenshot clearing cookies from a single website in chrome]
![clear-cookies](https://user-images.githubusercontent.com/1466013/72724585-a4325680-3b7b-11ea-98b9-46d2b86eabeb.gif)
